### PR TITLE
Bump to NVML version 11.3

### PR DESCRIPTION
**Pull request by klueska**

### CODE REVIEW

File: gen/nvml/device.go [hunk 1]
```diff
@@ -1256,6 +1256,24 @@ func (Device Device) ResetGpuLockedClocks() Return {
 	return DeviceResetGpuLockedClocks(Device)
 }
 
+// nvmlDeviceSetMemoryLockedClocks()
+func DeviceSetMemoryLockedClocks(Device Device, MinMemClockMHz uint32, MaxMemClockMHz uint32) Return {
```
Comment (elezar) [inline gen/nvml/device.go:1260]: I have probably asked this before, but are we explicitly using `uint32` here for `unsigned int` in C?

File: gen/nvml/init.go [hunk 2]
```diff
@@ -93,6 +93,10 @@ var nvmlDeviceGetAttributes = nvmlDeviceGetAttributes_v1
 var nvmlComputeInstanceGetInfo = nvmlComputeInstanceGetInfo_v1
 var nvmlDeviceGetComputeRunningProcesses = nvmlDeviceGetComputeRunningProcesses_v1
 var nvmlDeviceGetGraphicsRunningProcesses = nvmlDeviceGetGraphicsRunningProcesses_v1
+var GetBlacklistDeviceCount = GetExcludedDeviceCount
```
Comment (elezar) [inline gen/nvml/init.go:96]: Just to be clear, this is just a rename to allow this to be backward compatible?

File: gen/nvml/nvml.h [hunk 3]
```diff
@@ -1,3 +1,5 @@
+/*** NVML VERSION: 11.3.58 ***/
```
Comment (elezar) [inline gen/nvml/nvml.h:1]: Just a note. A really like this addition.

**Review by elezar — APPROVED** [review]
LGTM

File: gen/nvml/device.go (hunk 1 above)
Comment (klueska) [reply gen/nvml/device.go:1260]: Only when it makes sense for the type. See the discussion here:
https://github.com/NVIDIA/go-nvml/issues/31

**Pull request merged**